modules/sbo-configuration-of-directory-path-to-project-binding-data.adoc

```diff
@@ -0,0 +1,68 @@
+[id="sbo-configuration-of-directory-path-to-project-binding-data_{context}"]
+= Configuration of the directory path to project the binding data inside workload container
+
+For projecting the binding data, by default, files are mounted at a specific directory in your workload resource. You can configure the directory path using the existing `SERVICE_BINDING_ROOT` environment variable setup in the container where your workload runs.
+
+.Example
+----
+$SERVICE_BINDING_ROOT <1>
+├── account-database <2>
+│   ├── type <3>
+│   ├── provider <4>
+│   ├── uri
+│   ├── username
+│   └── password
+└── transaction-event-stream <2>
+    ├── type
+    ├── connection-count
+    ├── uri
+    ├── certificates
+    └── private-key
+----
+<1> Root directory.
+<2> Directory that stores the binding data.
+<3> Mandatory identifier that identifies the type of the binding data projected into the corresponding directory.
+<4> An optional identifier to identify the provider so that the application can identify the type of  backing service it can connect to.
```

Comment:
```suggestion
<4> An optional identifier to identify the provider so that the application can identify the type of backing service it can connect to.
```

Reply:
Consider adding or removing articles to the beginnings of these callouts.

Reply:
WIll change it to: "Optional identifier ...."

Reply:
Please search our documentation to see how `Optional:` is the preferred form.